etc/calico/confd/templates/bird_ipam.cfg.template

```diff
@@ -11,6 +21,16 @@ filter calico_pools {
 
 {{$network_key := printf "/bgp/v1/host/%s/network_v4" (getenv "NODENAME")}}{{if exists $network_key}}{{$network := getv $network_key}}
 filter calico_ipip {
+{{- if ls $static_key}}
+
+  # Don't program static routes into kernel.
```

Comment:
Do we know what might happen if node-to-node mesh is enabled and we learn a static route from another node?

I'm guessing right now, we will program it into the kernel. We probably need to avoid that (either by only doing this when node-to-node mesh, or ideally somehow ignoring static routes from other nodes independent of how they are peered).

Reply:
Not sure I'm following.  I believe the config above _will_ prevent programming static routes into the kernel, regardless of how they are learnt.

Are you thinking of when the static route set is different on different nodes?

Reply:
Yeah that's exactly the case I was thinking - when the static routes advertised by one node are different from another.

Reply:
Cool; I will address this point when I try to write up the UX for approval, but will leave the code as is for now.  (I suspect it's already as sensible as possible, if different static routes are configured on different nodes.)